garage/tf/algos/npo.py

```diff
@@ -76,35 +76,46 @@ def init_opt(self):
             else:
                 valid_var = None
 
-            dist_info_vars = self.policy.dist_info_sym(obs_var,
-                                                       state_info_vars)
-            kl = dist.kl_sym(old_dist_info_vars, dist_info_vars)
-            lr = dist.likelihood_ratio_sym(action_var, old_dist_info_vars,
-                                           dist_info_vars)
-            if is_recurrent:
+        dist_info_vars = self.policy.dist_info_sym(obs_var, state_info_vars)
+        kl = dist.kl_sym(old_dist_info_vars, dist_info_vars)
+        lr = dist.likelihood_ratio_sym(action_var, old_dist_info_vars,
+                                       dist_info_vars)
+        mean_kl_scope = tf.name_scope("mean_kl", values=[kl, valid_var])
```

Comment:
this could be made much easier to read (eliminating the `mean_kl_scope` and `surr_loss_scope` variables) by splitting the blocks into a mean_kl block and a surr_loss block.

Reply:
Done